models/os_image.go

```diff
@@ -0,0 +1,132 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package models
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/go-openapi/validate"
+)
+
+// OsImage os image
+//
+// swagger:model os-image
+type OsImage struct {
```

Comment:
I think this would still be better if it matched the [field names used in the k8s API](https://github.com/openshift/assisted-service/blob/72eb9bb/api/v1beta1/agentserviceconfig_types.go#L27-L37). Any reason not to do that?

FWIW part of the reasoning for the naming there avoiding "RHCOS" is that this could all theoretically be used with OKD and fedora coreos.

Reply:
To be clear, I'm not talking about case. I'm talking about the names of all the fields below that are part of this struct. They are very different from the names of the same fields in the k8s API struct.

Reply:
oh ok, kept the old keys just for an easier transition to the new format. Changed to correlate with k8s API.